scanner/combiner.go

```diff
@@ -41,6 +50,13 @@ func (c *Combiner) Read() (*zng.Record, error) {
 				c.done[k] = true
 				continue
 			}
+			mapper := c.mappers[k]
+			id := tup.Type.ID()
```

Comment:
(we'll need to get rid of tup at some point...)

Reply:
agreed